spec/s3_meta_sync_spec.rb

```diff
@@ -154,6 +156,12 @@ def self.it_downloads_into_an_empty_folder
         no_cred_syncer.sync("#{config[:bucket]}:bar", "foo")
         File.exist?("foo/zzz").should == false
       end
+    
+      it "passes by local files that no longer exist but described in .s3-meta-sync" do
+	`echo foo/wrong: 6cf4485094745f1e1815eb2c7ef46d5c >> .s3-meta-sync`
```

Comment:
use `sh "echo foo/wrong: 6cf4485094745f1e1815eb2c7ef46d5c >> .s3-meta-sync"` so it blows up if it fails or there is a typo